src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Resources/config/services.yml

```diff
@@ -255,6 +255,8 @@ services:
             - '@logger'
             - '%playground_uri_test%'
             - '%playground_uri_prod%'
+            - '%manage_prod_publication_status_update%'
```

Comment:
Instead of injecting the params directly, you could have used the Manage Config VO ®.

Reply:
More importantly, the param values are the other way aroud, the prod value is set for the test entities and vise versa.